src/System.Data.SqlClient/ref/System.Data.SqlClient.Manual.cs

```diff
@@ -7,6 +7,25 @@
 
 namespace System.Data.SqlClient
 {
+    // Declaring members from stripped base class CollectionBase
+    public sealed partial class SqlBulkCopyColumnMappingCollection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
Another thing to think about: if `SqlBulkCopyColumnMappingCollection` were a new API being added to CoreFX today, it'd just derive from `System.Collections.ObjectModel.Collection<T>` which does not have a public `Capacity` property.